### DISCUSSION TITLE
Welcome to pupnp Discussions!

### DISCUSSION THREAD

**Discussion by mrjimenez** [pupnp/pupnp#208] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by whyman** [comment]
Time to kill of the sourceforge project yet? :smile:

**Reply by mrjimenez** [reply]
I have no problem with that.

We currently use the site only for tarball distribution, but Github can easily do the job.

Also we need to put up the web site.
